src/NRedisStack/Gears/GearsCommands.cs

```diff
@@ -0,0 +1,65 @@
+using StackExchange.Redis;
+namespace NRedisStack
+{
+
+    public static class GearsCommands //: GearsCommandsAsync, IGearsCommands
+    {
+
+        /// <summary>
+        /// Load a new library to RedisGears.
+        /// </summary>
+        /// <param name="libraryCode">the library code.</param>
+        /// <param name="config">a string representation of a JSON object
+        /// that will be provided to the library on load time,
+        /// for more information refer to
+        /// <see href="https://github.com/RedisGears/RedisGears/blob/master/docs/function_advance_topics.md#library-configuration">
```

Comment:
Find out from Meir what the redis.io/commands/<name> will be for these, and let's update them all. Having multiple <see> references seems like a good idea